src/couch/test/fixtures/os_daemon_bad_perm.sh

```diff
@@ -15,3 +15,7 @@
 # Please do not make this file executable as that's the error being tested.
 
 sleep 5
+
```

Comment:
uh....why are you modifying this file? It's not supposed to run at all, see the comment on L15 :)

Reply:
I modified then reverted it but it left whitespace changes. Will fix